java/pom.xml

```diff
@@ -425,11 +386,14 @@
         <dynamodb.profile.name>dynamodb</dynamodb.profile.name>
       </properties>
       <dependencies>
+        <dependency>
```

Comment:
Remove this as won't build with it in currently

Reply:
actually the build was succesfull  if it put inside a profile